README.md

```diff
@@ -303,7 +303,7 @@ See [here](https://openvinotoolkit.github.io/openvino_notebooks/?search=Automati
 
 ## Additional materials
 
-- [List of supported models](https://github.com/openvinotoolkit/openvino.genai/blob/master/src/docs/SUPPORTED_MODELS.md) (NOTE: models can work, but were not tried yet)
+- [List of supported models](./src/docs/SUPPORTED_MODELS.md) (NOTE: models can work, but were not tried yet)
```

Comment:
This README is included in PyPI description. We need the link to github

Reply:
ok, see https://github.com/openvinotoolkit/openvino.genai/pull/1503